src/components/Article.jsx

```diff
@@ -0,0 +1,45 @@
+import { useParams } from 'react-router-dom'
+import {useEffect, useState} from 'react'
+import axios from 'axios'
+
+function Article({articles}){
```

Comment:
Why are you sending articles on props if you're going to do an api call in this component? There is no need for this to be here.